README.md

```diff
@@ -0,0 +1,49 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> The goal for this project is to master all of the tools and best practices we learned about in previous steps, such as GitHub flow, linter configuration.
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+To get a local copy for this project and running follow these simple example steps.
+
+### Prerequisites
+
+- You need to have git installed in your machine.
+
+### Setup
+
+- To clone the repository run this command `git clone https://github.com/j-tee/my-portfolio.git`
+
+
+## Author
+
+👤 **TeeJay**
+
+- GitHub: [@https://github.com/j-tee)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/j-tee/my-portfolio.git/issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is MIT licensed.
```

Comment:
- Please, kindly add the licensed content and it should take users to the content.

Reply:
Hi, Please elaborate on this a little more for me